zaza/utilities/deployment_env.py

```diff
@@ -116,12 +118,7 @@ def is_valid_env_key(key):
     :returns: Whether key is a valid environment variable name
     :rtype: bool
     """
-    valid = False
-    for _k in VALID_ENVIRONMENT_KEY_PREFIXES:
-        if key.startswith(_k):
-            valid = True
-            break
-    return valid
+    return key.startswith(VALID_ENVIRONMENT_KEY_PREFIXES)
```

Comment:
Nice optimisation; might be useful to add a comment indicating that `.startswith()` can also take the tuple of values to match against (which I didn't know about!  - TIL!)